tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -209,6 +209,17 @@ Verify CPU And Memory Requests And Limits Are Defined For All Containers In All
     Verify CPU And Memory Requests And Limits Are Defined For All Containers In All Pods In Project    redhat-ods-monitoring
     Verify CPU And Memory Requests And Limits Are Defined For All Containers In All Pods In Project    redhat-ods-operator
 
+Verify Monitoring Stack Is Reconciled Without Restarting The ODS Operator
+    [Documentation]    Verify Monitoring Stack Is Reconciled Without Restarting The RHODS Operator
+    [Tags]    Sanity
+    ...       Tier2
+    ...       ODS-699
+    OpenShiftCLI.Patch    kind=ServiceMonitor
+    ...                   src={"spec":{"selector":{"matchLabels": {"app":"grafana"}}}}
+    ...                   name=rhods-monitor-federation   namespace=redhat-ods-monitoring  type=merge
```

Comment:
you could wrap this into `Replace "Prometheus" with "Grafana" in Rhods-Monirot-Federation` to be consistent with line 221 and more self-explanatory

Reply:
done